src/components/maps/MapOverviewCard.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+    <div class="map-preview" @click="mapSelected">
+        <div class="background" :style="`background-image: url('${mapImageUrl}')`" />
+        <div class="header">
+            <div class="title">
+                <div class="flex-row flex-center gap-sm">
+                    {{ map.friendlyName }}
+                </div>
+            </div>
+        </div>
+
+        <div class="container">
+          <div class="thumbnail">
+            <BattleMapPreview
+                :map="map.scriptName"
+            />
+          </div>
+          <div class="details">
+            <div class="detail-text"><b>Size:</b> {{map.width}} x {{map.height}}</div>
+            <div class="detail-text"><b>Wind:</b> {{map.minWind}} - {{map.maxWind}}</div>
+            <div class="detail-text"><b>Description:</b> {{map.description}}</div>
+            <div class="detail-text"><b>Author:</b> {{map.mapInfo.author}}</div>
```

Comment:
This breaks rendering when the user has old maps installed that don't have `mapInfo.lua` files (which means `map.mapInfo` here is null)